File: golang/2180. 统计各位数字之和为偶数的整数个数.go
func countEven(num int) (ans int) {
    for i := 1; i <= num; i++ {
        sum := 0
        for x := i; x > 0; x /= 10 {
            sum += x % 10
        }
        if sum%2 == 0 {
            ans++
        }
    }
    return
}

File: 刷题/cpp/2180. 统计各位数字之和为偶数的整数个数.cpp
class Solution {
public:
    int countEven(int num) {
        int ans=0;
        for(int i=1;i<=num;i++){
            int x=i;
            int sum=0;
            while(x){
                sum+=x%10;
                x/=10;
            }
            if((sum&1)==0){
                ans++;
            }
        }
        return ans;
    }
};